src/node-agent/pkg/service/env.go

```diff
@@ -0,0 +1,51 @@
+package service
+
+import (
+	"github.com/sirupsen/logrus"
+	"os"
+)
+
+var (
+	podName           string
+	podNamespace      string
+	nodeName          string
+	printHttpRequests = false
+)
+
+func init() {
```

Comment:
we typically use `viper` rather than raw env vars, which provides better control over argument types etc. Consider switching to it, both for consistency with other portions of our application, and for better code. you can take a look at the sniffer code for a relatively simple example.